tests/CG/Tests/Generator/Fixture/generated/php7_class.php

```diff
@@ -0,0 +1,61 @@
+namespace CG\Tests\Generator\Fixture;
```

Comment:
From what I could tell, none of the generated files had opening PHP tags. It seems that this library inserts the opening PHP tag in the generator itself - the Enhancer class, for example, seems to make the assumption that it has to insert the start tag. Am I making a fundamental mistake in my understanding of your structure?